templates/_layouts/examples.html

```diff
@@ -21,6 +21,7 @@
         <link rel="stylesheet" type="text/css" href="{{ versioned_static('build/css/docs/example.css') }}" />
         <link rel="icon" href="https://assets.ubuntu.com/v1/ab36e6ed-vanilla_favicon_32px.png" type="image/x-icon" />
       {% block style %}{% endblock %}
+      {% block script %}{% endblock %}
```

Comment:
Our scripts don't usually wait for DOM to be loaded or anything, and are placed at the end of the body, so I'd suggest doing the same here. Just move it down right before body closing tag.